build.gradle.kts

```diff
@@ -114,12 +114,18 @@ sonar {
                 "dataland-frontend/src/frameworks/heimathafen/UploadConfig.ts," +
                 "dataland-frontend/src/frameworks/custom/EuTaxoNonFinancialsStaticUploadConfig.ts," +
                 "dataland-frontend/src/frameworks/sfdr/UploadConfig.ts," +
+                "dataland-frontend/src/frameworks/lksg/ViewConfig.ts," +
+                "dataland-frontend/src/frameworks/lksg/UploadConfig.ts," +
                 "dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgDataModel.ts," +
                 "dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts," +
                 "dataland-frontend/src/components/resources/frameworkDataSearch/sme/SmeDataModel.ts," +
 
                 // backend
-                "dataland-backend/src/main/kotlin/db/migration/V1_1__CreateBackendTables.kt",
+                "dataland-backend/src/main/kotlin/db/migration/V1_1__CreateBackendTables.kt," +
+
+                // toolbox
+                "dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/components" +
```

Comment:
This is not generated, thus I don't like it being added here. What is the reason?

Reply:
apart from the removed possibility to pass options from the excel this is identical to the multi select component.
if i were to handle this in the multi select itself, i would have to check for a certain string in options which is kind of messy. therefore i just created a new component, which definitely has a use case in the future. 
similar to nace codes which are a special type of multi select

Reply:
What is the sonar issue you are trying to circumvent? If it is duplication, woult it be possible to let this component derive from the multiselectcomponent?

Reply:
yeah it's duplication.
But in my opinion this creates messy code for the multi select component and contradicts the toolboxes code structure. i am happy to show you that

Reply:
Nope. Fine. I feared so already.